src/components/role/index.js

```diff
@@ -1,22 +1,24 @@
-import React, {PropTypes} from 'react';
+import React, {PropTypes, Component} from 'react';
 
 import user from 'focus-core/user';
 import intersection from 'lodash/array/intersection';
 import isArray from 'lodash/lang/isArray';
 
-function Role({hasAll,hasOne,children,emptyBlock}) {
-    let userRoles = user.getRoles();
-    //console.log("emptyBlock",emptyBlock);
-    if(isArray(hasAll) && intersection(userRoles, hasAll).length === hasAll.length) {
-        return children;
-    } else if(isArray(hasOne) && intersection(userRoles, hasOne).length > 0) {
-        return children;
+class Role extends Component { 
```

Comment:
NO : "A stateless component can't return null or false"

So, as a function the component doesn't work.